css/components/term.css

```diff
@@ -1,21 +1,21 @@
-.lgd-term--content__image img {
+.lgd-term--image img {
```

Comment:
We need to fix up the CSS classes here to reflect the BEM classes that we will use in the HTML. `lgd-term__image` instead of `lgd-term--image`, etc.